moto/iam/responses.py

```diff
@@ -484,9 +484,9 @@ def get_user(self):
                 user = User("default_user")
         else:
             user = iam_backend.get_user(user_name)
-
+            tags = iam_backend.tagger.list_tags_for_resource(user.arn).get("Tags", [])
```

Comment:
Looks like the indentation is incorrect - it shouldn't be part of the `else` statement